inlineplz/interfaces/github.py

```diff
@@ -67,16 +68,19 @@ def is_duplicate(self, message, position):
         return False
 
     @staticmethod
-    def format_message(message):
+    def format_message(message, prefix):
         if not message.comments:
             return ''
         if len(message.comments) > 1:
-            return (
-                '```\n' +
-                '\n'.join(sorted(list(message.comments))) +
-                '\n```'
-            )
-        return '`{0}`'.format(list(message.comments)[0].strip())
+            comment_output = '\n'.join(sorted(list(message.comments)))
+            if prefix:
+                comment_output = prefix + '\n' + comment_output
+            return '```\n {0} \n ```'.format(comment_output)
+
+        comment_output = list(message.comments)[0].strip()
```

Comment:
Considered moving this to s function since it is more or less duped code, but given its just 2 lines each, and this is already the 'format message' function, opted to leave it here. Open to changing that.